grpc/sa-wrappers.go

```diff
@@ -516,6 +516,18 @@ func (sas StorageAuthorityClientWrapper) GetOrder(ctx context.Context, request *
 	return resp, nil
 }
 
+func (sas StorageAuthorityClientWrapper) GetOrderForNames(
+	ctx context.Context,
+	request *sapb.GetOrderForNamesRequest) (*corepb.Order, error) {
+	resp, err := sas.inner.GetOrderForNames(ctx, request)
+	if err != nil {
+		return nil, err
+	}
+	// We do not check `orderValid(resp)` here because an "empty" order response
```

Comment:
In other RPCs we return NotFoundError for this case; any reason not to do it here?

Reply:
No reason except I didn't think of that. I think that's a better approach and will also [address Roland's comment](https://github.com/letsencrypt/boulder/pull/3290#discussion_r159310736) since I can restore the `if resp != nil -> make sure order is valid` clause.